code/modules/admin/verbs/grief_fixers.dm

```diff
@@ -0,0 +1,54 @@
+/client/proc/fixatmos()
+	set category = "Admin"
+	set name = "Fix Atmospherics Grief"
+
+	if(!check_rights(R_ADMIN|R_DEBUG)) return
+
+
+	if(alert("WARNING: Executing this command will perform a full reset of atmosphere. All pipelines will lose any gas that may be in them, and all zones will be reset to contain air mix as on roundstart. The supermatter engine will also be stopped (to prevent overheat due to removal of coolant). Do not use unless the station is suffering serious atmospheric issues due to grief or bug.", "Full Atmosphere Reboot", "No", "Yes") == "No")
+		return
+	feedback_add_details("admin_verb","FA")
+
+	log_and_message_admins("Full atmosphere reset initiated by [usr].")
+	world << "<span class = 'danger'>Initiating restart of atmosphere. The server may lag a bit.</span>"
+	sleep(10)
```

Comment:
I don't see a problem here, can you elaborate, assuming you mean the sleep() call?